extensions/roc-package-web-app-react/app/default/get-routes-and-store.js

```diff
@@ -56,15 +56,22 @@ export default function getRoutesAndStore() {
         store = storeCreator(replaceReducers);
     }
 
+    const { default: projectRoutes, middlewareConfig = {} } = require(REACT_ROUTER_ROUTES);
```

Comment:
Better error messages are a great idea, but as you say not part of this PR scope.